docs/source/docker/advanced/sequence_selection.rst

```diff
@@ -0,0 +1,80 @@
+Sequence Selection
+==================
+
+Sequence selection allows users to select sequences of a video instead of single frames.
+The key concept is the parameter `selected_sequence_length`. If its value is one (default),
+the docker selects single frames. If it is larger than one, each video is split into 
+sequences of that length and the frame representations are aggregated into a sequence
+representation. The sampling then happens on these sequence representations.
+
+.. note:: Sequence selection works on videos or on folders of alphabetically sorted
+    frames.
+
+
+- | If you're interested in how the sequence selection works, go to
+  | --> `How It Works`_
```

Comment:
Not sure this helps as the whole page is super small. Adding links to jump make more sense if you need to scroll a lot.